index.html

```diff
@@ -22,14 +22,15 @@
   <header>
     <div class="container">
       <div class="row">
-        <div class="col-full">
-          <h2>Code Journal</h2>
+        <div class="col-full nav">
+          <h3>Code Journal</h3>
+          <a id="entries-link">Entries</a>
         </div>
       </div>
     </div>
   </header>
   <main class="container">
-    <div data-view="entry-form">
+    <div data-view="entry-form" class="hidden">
```

Comment:
The `entry-form` should be visible by default. So it shouldn't have a class of `hidden`